### ISSUE TITLE
json dependency unsatisfiable

### ISSUE THREAD

**Issue by syphernl** [vulogov/zq#7]
I'm trying to install this on an Ubuntu 17.04 box, but installing the `json` dependency doesn't seem possible:
```
Collecting json (from zq)
  Could not find a version that satisfies the requirement json (from zq) (from versions: )
No matching distribution found for json (from zq)
```

**Comment by vulogov** [comment]
What is the version of the Python ? If 3.0, that it is not supported (by me) yet. json module is the part of standard library since 2.6

**Comment by vulogov** [comment]
https://docs.python.org/2/library/json.html

**Comment by nevill** [comment]
I have the same issue on macOS 10.13 with python 2.7.13 (via miniconda). `json` is a built-in module, cannot be installed with things like `pip install json`

After I removed [`json` from setup.py](https://github.com/vulogov/zq/blob/master/setup.py#L61), I can do `pip install .` locally with success.

**Comment by vulogov** [comment]
Thank you. I was wrong.

**Comment by vulogov** [comment]
This bug is closed in 0.7-RC2

https://github.com/vulogov/zq/tree/0.7-RC2
